README.md

```diff
@@ -1,56 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# To Do list with Webpack
 
+> n this project, I build a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.
 
```

Comment:
- [ ] There is a typo error at the beginning of this sentence, you have `n` but I guess you wanted to type `In`. Kindly fix it.